### ISSUE TITLE
Change display resolution settings

### ISSUE THREAD

**Issue by Bajiio** [zaproxy/zaproxy#7684]
Hello,
I just downloaded ZAP on my new laptop and I have a display resolution problem :  everything is very small and it's really annoying to click on icons, etc...
I'm on Ubuntu 20.04 LTS, my screen resolution is 2880x1800 (16:10). ```java -version``` show "11.0.17".

Here's how it looks : 
![image](https://user-images.githubusercontent.com/60874928/211794331-c914415c-9fe5-40c6-8f15-895c42c41771.png)

I already changed the font size, which helped a bit but didn't fix the icons in the top level bar.
I also tried to add  ```-Dsun.java2d.uiscale=2.5``` in my zap.sh file but it didn't change anything.

How can I fix that ? 

Thanks in advance for your help !

**Comment by thc202** [comment]
Closing as duplicate of #7326.

**Comment by thc202** [comment]
Although the icons should be increased with the font, did you change the font size in ZAP?

**Comment by Bajiio** [comment]
Yes I did, I changed the general font size and the Work Panels Font

**Comment by thc202** [comment]
OK, so this is other issue.

**Comment by Bajiio** [comment]
![image](https://user-images.githubusercontent.com/60874928/211856065-99ba7d32-a041-498f-bb37-5b3f6d7e0e1d.png)
Here's my display settings in ZAP if that can help someone

**Comment by thc202** [comment]
Ah, you need to check Scale Images for them to be changed along the font size.

**Comment by Bajiio** [comment]
Ok it seems I could see the problem, the font size is not changing anymore. Here the font is supposed to be at size 90 but everything is normal. So I guess icons are not scaling with font size because font size doesn't change.
![image](https://user-images.githubusercontent.com/60874928/211880237-e960e602-71ba-4d62-852a-230d5821e66d.png)

**Comment by psiinon** [comment]
You need to restart ZAP before any font size changes take effect.

**Comment by Bajiio** [comment]
Oh ok I didn't know sorry. I just did and yes, icons scale with text but it's still not usable to be honest.. Is there any way to make the top level bar bigger ? We can't even see the "standard mode" correctly.

**Comment by psiinon** [comment]
Can you share another screenshot? "not usable" doesnt realy tell us much 😉

**Comment by Bajiio** [comment]
Here's with Font size = 35
![image](https://user-images.githubusercontent.com/60874928/212321117-cd73fc7b-7df0-4882-adee-60817fb4b431.png)
And now with a size of 60, the icon size is now correct, but text size is annoying to work with
![image](https://user-images.githubusercontent.com/60874928/212321467-62379047-6911-4014-81aa-fa0396b72d02.png)
The ratio between text and icon is the same, and that is what I look for to change, so I can only modify icon size, or top level bar size.

**Comment by psiinon** [comment]
Are you _sure_ you have checked "scale images" before restarting?
This is a screenshot of ZAP with the General Font set to 30 for me with scale images set:
<img width="1530" alt="Screenshot 2023-01-16 at 10 29 15" src="https://user-images.githubusercontent.com/1081115/212657279-28a06228-0ad1-42a3-a360-926c3e42db98.png">
As you can see the icons are significantly bigger.

**Comment by Bajiio** [comment]
Yes I am sure. I don't why It has this problem, to be honest I started using it from a VM and I don't have this problem at all.

**Comment by psiinon** [comment]
I dont know why either :/
Can you share a screenshot of your display options as they are right now just in case we can spot anything?

**Comment by psiinon** [comment]
I cant reproduce that on my MacBook 😦 
Can anyone  else reproduce it?
If we can't reproduce it we won't be able to test any potential fixes...

**Comment by ia-mroot** [comment]
I fixed it using the display settings in the app. Here is my config and you need to restart the application after the changes for them to apply.

<img width="1131" height="703" alt="Image" src="https://github.com/user-attachments/assets/c86ea5cd-0a69-4011-b393-66e8453f3f93" />

Hope this helps.
